<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantWidget</class>
 <widget class="QWidget" name="AssistantWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>助教信息</string>
  </property>
  <widget class="QListWidget" name="listStudent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>230</width>
     <height>281</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnrollAssist">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>报名助教</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelectAssist">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择助教</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCourse">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>课程信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurAssistInfo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前助教</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAssistList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>助教名单</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAssist">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnEnrollAssist</sender>
   <signal>clicked()</signal>
   <receiver>AssistantWidget</receiver>
   <slot>enrollAssistant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelectAssist</sender>
   <signal>clicked()</signal>
   <receiver>AssistantWidget</receiver>
   <slot>selectAssistant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enrollAssistant()</slot>
  <slot>selectAssistant()</slot>
 </slots>
</ui>
